ts/features/wallet/common/BasePaymentMethodScreen.tsx

```diff
@@ -1,136 +1,175 @@
+import { ScrollView } from "react-native-gesture-handler";
+import { widthPercentageToDP } from "react-native-responsive-screen";
+import LoadingSpinnerOverlay from "../../../components/LoadingSpinnerOverlay";
+import { useIOToast } from "../../../components/Toast";
 import { VSpacer } from "../../../components/core/spacer/Spacer";
-import { Label } from "../../../components/core/typography/Label";
 import { IOColors } from "../../../components/core/variables/IOColors";
+import { IOSpacingScale } from "../../../components/core/variables/IOSpacing";
 import { IOStyles } from "../../../components/core/variables/IOStyles";
-import LoadingSpinnerOverlay from "../../../components/LoadingSpinnerOverlay";
-import DarkLayout from "../../../components/screens/DarkLayout";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import FocusAwareStatusBar from "../../../components/ui/FocusAwareStatusBar";
+import ListItemAction from "../../../components/ui/ListItemAction";
 import I18n from "../../../i18n";
-import { navigateBack } from "../../../store/actions/navigation";
 import { deleteWalletRequest } from "../../../store/actions/wallet/wallets";
-import { GlobalState } from "../../../store/reducers/types";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
 import { getWalletsById } from "../../../store/reducers/wallet/wallets";
 import { PaymentMethod } from "../../../types/pagopa";
 import { emptyContextualHelp } from "../../../utils/emptyContextualHelp";
-import { showToast } from "../../../utils/showToast";
 import { useRemovePaymentMethodBottomSheet } from "../component/RemovePaymentMethod";
 
-type OwnProps = {
+type Props = {
   paymentMethod: PaymentMethod;
   card: React.ReactNode;
   content: React.ReactNode;
 };
 
-type Props = ReturnType<typeof mapDispatchToProps> &
-  ReturnType<typeof mapStateToProps> &
-  OwnProps;
+// ----------------------- card layout calculations -----------------------
+
+// base consts
+const CARD_WIDTH = widthPercentageToDP(100);
+const CARD_HEIGHT = 0.5 * CARD_WIDTH;
+
+// the amount of header to render in case of scrollDown
+const absoluteMarginTopHeight = 2 * CARD_HEIGHT;
+
+// "progressbar-like" percentage of card that has a
+// blue BG, can be played around with
+const percentageOfCardWithBackground = 75;
+
+// the actual height of the visible (without scrolling) card's background
+const cardBackgroundHeight =
+  (percentageOfCardWithBackground / 100) * CARD_HEIGHT;
+
+// how much the card overflows under the header, used as absolute positioning
+// and as bottom spacer
+const cardOverflowAmount = (100 - percentageOfCardWithBackground) * 2;
+
+// the header's actual height, including the overflow that can only be seen
+// when scrolling downwards
+const headerHeight = absoluteMarginTopHeight + cardBackgroundHeight;
+
+// ----------------------------- styles -----------------------------------
```

Comment:
```suggestion
```

There is not need of this.
You could solve the problem just by using styles:

```
const styles = StyleSheet.create({
  cardContainer: {
    paddingHorizontal: 16,
    alignItems: "center",
    marginBottom: "-15%"
  },
  blueHeader: {
    backgroundColor: IOColors["blueIO-600"],
    paddingTop: "75%",
    marginTop: "-75%",
    marginBottom: "15%"
  }
});
```

Try this.

Reply:
well yes, you're right 😅 
adressed nonetheless 👍